curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67f90574263ebf363f9edd1d.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-15
 
 If `query` is not empty, you want to get fruits that match the user input from the API, but only after users stop typing for a short period to avoid making the fetch call too frequently.
 
-To start, after the `if` statement, call `setTimeout` with a delay value of `700` and store it in a variable called `timeoutId`. This allows you to cancel the timeout later when the effect cleans up.
+To start, after the `if` statement, call `setTimeout` with an empty arrow function (`()=>{}`) and a delay value of `700` as arguments, and store it in a variable called `timeoutId`. This allows you to cancel the timeout later when the effect cleans up.
```

Comment:
```suggestion
To start, after the `if` statement, call `setTimeout` with an empty arrow function and a delay value of `700` as arguments, and store it in a variable called `timeoutId`. This allows you to cancel the timeout later when the effect cleans up.
```

it is advanced enough in the curriculum that campers already know what it looks like